src/wxProfilerGUI/profilergui.cpp

```diff
@@ -728,18 +728,13 @@ bool ProfilerGUI::OnCmdLineParsed(wxCmdLineParser& parser)
 	if (parser.Found("a", &param))
 		cmdline_attach = param.c_str();
 	if (parser.Found("thread", &param)) {
-		std::cout << "-thread found (" << param << ")\n";
 		auto tids_str = wxSplit(param,',');
 		for (size_t i=0; i<tids_str.GetCount(); i++) {
 			long tid;
 			if (tids_str[i].ToLong(&tid)) {
 				cmdline_thread_ids.push_back(tid);
 			}
 		}
-		std::cout << "  Threads to attach to (";
-		for (auto& tid : cmdline_thread_ids)
-			std::cout << " " << tid;
-		std::cout << " )\n";

```

Comment:
Please squash this commit into the commit which added the debug code, so that it won't be present in the Git history.